<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Window</class>
 <widget class="QWidget" name="Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>773</width>
    <height>495</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Window</string>
  </property>
  <property name="windowIcon">
   <iconset resource="application.qrc">
    <normaloff>:/images/seo-panda-icon.png</normaloff>:/images/seo-panda-icon.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="5">
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <property name="sizeConstraint">
      <enum>QLayout::SetMaximumSize</enum>
     </property>
     <item>
      <widget class="MyGLWidget" name="myGLWidget" native="true">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>200</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>CrossCursor</cursorShape>
       </property>
      </widget>
     </item>
     <item>
      <widget class="widget3d" name="widget3D" native="true">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>200</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="5">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_10">
           <item>
            <widget class="QRadioButton" name="linea_button">
             <property name="text">
              <string>Línea</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="triangulo_button">
             <property name="text">
              <string>Triángulo</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="rectangulo_button">
             <property name="text">
              <string>Rectángulo</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="elipse_button">
             <property name="text">
              <string>Elipse</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioButton">
             <property name="text">
              <string>Bezier</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_12">
           <item>
            <widget class="QSlider" name="red_slider">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="autoFillBackground">
              <bool>false</bool>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: red</string>
             </property>
             <property name="maximum">
              <number>255</number>
             </property>
             <property name="value">
              <number>255</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="tickPosition">
              <enum>QSlider::NoTicks</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="green_slider">
             <property name="styleSheet">
              <string notr="true">background-color: green</string>
             </property>
             <property name="maximum">
              <number>255</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="blue_slider">
             <property name="styleSheet">
              <string notr="true">background-color: blue</string>
             </property>
             <property name="maximum">
              <number>255</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QProgressBar" name="progressBar">
           <property name="value">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spin_box"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QToolButton" name="toolButton_6">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="application.qrc">
             <normaloff>:/images/open.png</normaloff>:/images/open.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton_5">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="application.qrc">
             <normaloff>:/images/save.png</normaloff>:/images/save.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton_7">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="application.qrc">
             <normaloff>:/images/Button-Add-icon.png</normaloff>:/images/Button-Add-icon.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="done">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="application.qrc">
             <normaloff>:/images/Button-Ok-icon.png</normaloff>:/images/Button-Ok-icon.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton_9">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="application.qrc">
             <normaloff>:/images/Button-Cancel-icon.png</normaloff>:/images/Button-Cancel-icon.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton_8">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="application.qrc">
             <normaloff>:/images/Button-Refresh-icon.png</normaloff>:/images/Button-Refresh-icon.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_14">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Y Rotation</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignJustify|Qt::AlignVCenter</set>
             </property>
             <property name="wordWrap">
              <bool>false</bool>
             </property>
             <property name="margin">
              <number>0</number>
             </property>
             <property name="indent">
              <number>0</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>X Rotation</string>
             </property>
             <property name="margin">
              <number>0</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Z Rotation</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Prueba</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_15">
           <item>
            <widget class="QSlider" name="rotYSlider">
             <property name="maximum">
              <number>360</number>
             </property>
             <property name="singleStep">
              <number>16</number>
             </property>
             <property name="pageStep">
              <number>15</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="tickPosition">
              <enum>QSlider::TicksAbove</enum>
             </property>
             <property name="tickInterval">
              <number>15</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="rotXSlider">
             <property name="maximum">
              <number>360</number>
             </property>
             <property name="singleStep">
              <number>16</number>
             </property>
             <property name="pageStep">
              <number>15</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="tickPosition">
              <enum>QSlider::TicksAbove</enum>
             </property>
             <property name="tickInterval">
              <number>15</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="rotZSlider">
             <property name="minimum">
              <number>-100</number>
             </property>
             <property name="maximum">
              <number>360</number>
             </property>
             <property name="singleStep">
              <number>16</number>
             </property>
             <property name="pageStep">
              <number>15</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="tickPosition">
              <enum>QSlider::TicksAbove</enum>
             </property>
             <property name="tickInterval">
              <number>15</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="myPrueba">
             <property name="cursor">
              <cursorShape>SplitHCursor</cursorShape>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="tickPosition">
              <enum>QSlider::TicksAbove</enum>
             </property>
             <property name="tickInterval">
              <number>1</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QToolButton" name="toolButton_10">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="application.qrc">
             <normaloff>:/images/Button-Info-icon.png</normaloff>:/images/Button-Info-icon.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton_3">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="application.qrc">
             <normaloff>:/images/new.png</normaloff>:/images/new.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="application.qrc">
             <normaloff>:/images/cut.png</normaloff>:/images/cut.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="abrir_archivo">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="application.qrc">
             <normaloff>:/images/open.png</normaloff>:/images/open.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton_2">
           <property name="acceptDrops">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="application.qrc">
             <normaloff>:/images/Actions-window-fullscreen-icon.png</normaloff>:/images/Actions-window-fullscreen-icon.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonFollowStyle</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton_11">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="application.qrc">
             <normaloff>:/images/Button-Turn-Off-icon.png</normaloff>:/images/Button-Turn-Off-icon.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MyGLWidget</class>
   <extends>QWidget</extends>
   <header location="global">myglwidget.h</header>
   <container>1</container>
   <slots>
    <slot>setXRotation(int)</slot>
    <slot>setZRotation(int)</slot>
    <slot>setYRotation(int)</slot>
    <slot>setMyPrueba(int)</slot>
    <slot>linea_button_pressed_slot(bool)</slot>
    <slot>triangulo_button_pressed_slot(bool)</slot>
    <slot>rectangulo_button_pressed_slot(bool)</slot>
    <slot>elipse_button_pressed_slot(bool)</slot>
    <slot>progress_bar_slot(int)</slot>
    <slot>open_file_slot()</slot>
    <slot>done_button_pressed_slot()</slot>
    <slot>add_button_pressed_slot()</slot>
    <slot>red_slider_slot(int)</slot>
    <slot>green_slider_slot(int)</slot>
    <slot>blue_slider_slot(int)</slot>
    <slot>spin_slot(int)</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>widget3d</class>
   <extends>QWidget</extends>
   <header>widget3d.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="application.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>linea_button</sender>
   <signal>toggled(bool)</signal>
   <receiver>myGLWidget</receiver>
   <slot>linea_button_pressed_slot(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>56</x>
     <y>238</y>
    </hint>
    <hint type="destinationlabel">
     <x>280</x>
     <y>162</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>triangulo_button</sender>
   <signal>toggled(bool)</signal>
   <receiver>myGLWidget</receiver>
   <slot>triangulo_button_pressed_slot(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>56</x>
     <y>271</y>
    </hint>
    <hint type="destinationlabel">
     <x>280</x>
     <y>162</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>elipse_button</sender>
   <signal>toggled(bool)</signal>
   <receiver>myGLWidget</receiver>
   <slot>elipse_button_pressed_slot(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>56</x>
     <y>332</y>
    </hint>
    <hint type="destinationlabel">
     <x>280</x>
     <y>162</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rectangulo_button</sender>
   <signal>toggled(bool)</signal>
   <receiver>myGLWidget</receiver>
   <slot>rectangulo_button_pressed_slot(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>56</x>
     <y>304</y>
    </hint>
    <hint type="destinationlabel">
     <x>280</x>
     <y>162</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>progressBar</sender>
   <signal>valueChanged(int)</signal>
   <receiver>myGLWidget</receiver>
   <slot>progress_bar_slot(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>286</x>
     <y>362</y>
    </hint>
    <hint type="destinationlabel">
     <x>280</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>abrir_archivo</sender>
   <signal>clicked()</signal>
   <receiver>myGLWidget</receiver>
   <slot>open_file()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>525</x>
     <y>413</y>
    </hint>
    <hint type="destinationlabel">
     <x>280</x>
     <y>119</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>done</sender>
   <signal>clicked()</signal>
   <receiver>myGLWidget</receiver>
   <slot>done_button_pressed_slot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>192</x>
     <y>393</y>
    </hint>
    <hint type="destinationlabel">
     <x>129</x>
     <y>110</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButton_7</sender>
   <signal>clicked()</signal>
   <receiver>myGLWidget</receiver>
   <slot>add_button_pressed_slot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>143</x>
     <y>398</y>
    </hint>
    <hint type="destinationlabel">
     <x>86</x>
     <y>148</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>red_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>myGLWidget</receiver>
   <slot>red_slider_slot(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>172</x>
     <y>228</y>
    </hint>
    <hint type="destinationlabel">
     <x>170</x>
     <y>185</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>green_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>myGLWidget</receiver>
   <slot>green_slider_slot(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>146</x>
     <y>258</y>
    </hint>
    <hint type="destinationlabel">
     <x>146</x>
     <y>179</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>blue_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>myGLWidget</receiver>
   <slot>blue_slider_slot(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>203</x>
     <y>286</y>
    </hint>
    <hint type="destinationlabel">
     <x>190</x>
     <y>187</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButton_6</sender>
   <signal>clicked()</signal>
   <receiver>myGLWidget</receiver>
   <slot>open_file_slot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>33</x>
     <y>399</y>
    </hint>
    <hint type="destinationlabel">
     <x>44</x>
     <y>185</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spin_box</sender>
   <signal>valueChanged(int)</signal>
   <receiver>myGLWidget</receiver>
   <slot>spin_slot(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>334</x>
     <y>355</y>
    </hint>
    <hint type="destinationlabel">
     <x>304</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
